app/utils/Example-Tags.ts

```diff
@@ -0,0 +1,7 @@
+//Json Object
+export const ExampleTags = {
+    "Subjects": [ "Mathe", "Deutsch", "Englisch", "Latein", "Physik", "Biologie", "Geschichte", "Erdkunde"
+    ],
+    "Users": [ "testuser", "Ash", "Markus", "Nils", "Theo" ],
+    "File Types": [ "Bild", "Dokument", "Audio", "Video" ]
+};
```

Comment:
Not sure why we need this file. Also please stick with out PascalCase convention.